fineract-provider/src/main/java/org/apache/fineract/infrastructure/campaigns/sms/service/SmsCampaignDropdownReadPlatformServiceImpl.java

```diff
@@ -81,10 +80,8 @@ public SmsCampaignDropdownReadPlatformServiceImpl(final SmsConfigUtils smsConfig
                     new ParameterizedTypeReference<Collection<SmsProviderData>>() {});
             smsProviderOptions = responseOne.getBody();
             if (!responseOne.getStatusCode().equals(HttpStatus.OK)) {
-                throw new ConnectionFailureException(hostName);
             }
         } catch (Exception e) {
-        	 throw new ConnectionFailureException(hostName);
```

Comment:
Swallowing exception. Please check if it's intentional.

Reply:
We removed this exception , when sms will be send at that time it will throw validation message for the same.